docs/source/notebooks/SMC2_gaussians.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>But here, IIUC, ArviZ is supposed to report low ESS, right? It makes sense since the sampler had difficulties with the high dimensionality of the data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/3981/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
yeah, here it show problems. In this example each run/chain is basically sampling from a different mode and then ESS is showing a very low value ~2 (two modes).